pegasus/sites.v3/hourofcode.com/views/hoc_events_map.haml

```diff
@@ -1,6 +1,5 @@
 %link{rel: "stylesheet", type: "text/css", href: "/css/map.css"}
 %link{href:'https://api.mapbox.com/mapbox-gl-js/v1.0.0/mapbox-gl.css', rel:'stylesheet'}
```

Comment:
Should this be moved up too? I'm not sure if this CSS is only for the map or not

Reply:
Good question.  It might also affect the "Event location" dropdown?  It appears to be unaffected by this move...

#### localhost:

<img width="594" alt="Screen Shot 2021-09-22 at 3 11 04 PM" src="https://user-images.githubusercontent.com/2205926/134286779-d88901bd-ba90-4eba-adcc-269dece6f09e.png">
 
#### production:

<img width="591" alt="Screen Shot 2021-09-22 at 3 09 56 PM" src="https://user-images.githubusercontent.com/2205926/134286824-78c9e205-a3a6-4b7d-b223-20ab31189cd6.png">

Reply:
Yeah looks good!